script/makefiles/cluster-kind.mk

```diff
@@ -6,6 +6,7 @@
 KUBE ?= ${HOME}/.kube
 CLUSTER_NAME ?= kubeapps
 ADDITIONAL_CLUSTER_NAME ?= kubeapps-additional
+IMAGE ?= kindest/node:v1.24.0
```

Comment:
Kind recommends using the very exact image tag for each Kind version.  However, as we can change it just setting `IMAGE` is not a big deal.  Perhaps this image also works out of the box.

https://github.com/kubernetes-sigs/kind/releases

> New Node images have been built for kind v0.12.0, please use these exact images (IE like ....

Reply:
Yep, I'll update to the exact image - easy to set in the Makefile and we can do what we like when we override (as I did to test out 1.21).